web/src/services/products.js

```diff
@@ -11,6 +11,7 @@ export const searchProducts = async (searchTerm) => {
 
 const getProduct = async (id) => {
 	const response = await axios.get(`${baseUrl}/product?id=${id}`)
+	console.log(response)
```

Comment:
console.log left behind. =)

Reply:
I resolved the issue, thanks for pointing that out.